### PR TITLE
React to change in test file structure

### PR DESCRIPTION
**Pull request by natemcmaster**
cc @pranavkm

### CODE REVIEW

File: test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ public class LocalizationTest
     {
         private static readonly string _applicationPath =
 #if DNX451
-            Path.GetFullPath(Path.Combine("..", "..", "..", "LocalizationWebsite"));
+            Path.GetFullPath(Path.Combine("..", "..", "..", "..", "test", "LocalizationWebsite"));
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs]: Does https://github.com/aspnet/Localization/blob/dev/test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs#L16 need to be updated?

Comment (natemcmaster) [inline test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs]: I'm a bit confused by your question. Are you asking if this change is needed at all? You linked to the exact line I am changing in this PR

Comment (pranavkm) [inline test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs]: Sorry, I meant https://github.com/aspnet/Localization/blob/dev/test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationSampleTest.cs#L16

Comment (natemcmaster) [inline test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs]: Ah yes, there's that one too.

**Comment by pranavkm** [comment]
:shipit: if this builds locally.

**Comment by Eilon** [comment]
@natemcmaster can you merge this or is it blocked on anything?

**Comment by natemcmaster** [comment]
Waiting on https://github.com/aspnet/Universe/pull/372

**Comment by Eilon** [comment]
@natemcmaster got it, thanks!

**Comment by natemcmaster** [comment]
aspnet/Universe#372 was updated so the relative paths don't need to be changed.

**Pull request closed without merging**